corehq/form_processor/utils.py

```diff
@@ -33,3 +35,11 @@ def _inner(*args, **kwargs):
         return _wrap(obj)
 
     return _inner
+
+
+def get_backend():
```

Comment:
this obviously doesn't scale too well. curious if you had any thoughts on this. as in how do we get the backend based on the interface? i'm thinking the best route would be to define a `get_backend` property on each of the interfaces that will decide what backend class it needs based on the processor backend (either `couch` or `sql`)